src/planscape/planscape/settings.py

```diff
@@ -68,6 +68,7 @@
     'rest_framework_gis',
     'rest_framework_simplejwt',
     'rest_framework.authtoken',
+    'attributes',
```

Comment:
OOC, is there an ordering convention we are to follow here?

Reply:
I'm not certain, but my impression is the apps starting from attributes and below are from our codebase whereas the apps above it are external.

I can change this into two lists.

Reply:
done.
There are two lists in alphabetical order - one for downloaded apps, one for planscape appes.